Pear/Cells/EditInterestsGroupsPromptsCollectionViewCell.swift

```diff
@@ -0,0 +1,131 @@
+//
+//  EditInterestsGroupsPromptsCollectionViewCell.swift
+//  
+//
+//  Created by Mathew Scullin on 4/23/22.
+//  Copyright © 2022 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+class EditInterestsGroupsPromptsCollectionViewCell: UICollectionViewCell {
+    
+    static let reuseIdentifier = "EditInterestsGroupsPromptsCollectionViewCell"
+    
+    // MARK: - Private View Vars
+    private let backDropView = UIView()
+    private let describtorLabel = UILabel()
+    private let fadeTableView = FadeWrapperView(
+        UITableView(frame: .zero, style: .grouped), fadeColor: .backgroundLightGreen
+    )
+    
+    // MARK: - Private Data Vars
+    private var tableViewContents = [Any]()
```

Comment:
🤢

Reply:
I feel... this was like a first stab there is a better way to do this I shall revisit which I think will make dequeuing cells cleaner as well